### PR TITLE
Finished step 13 for mochawesome reports install and configuration

### PR DESCRIPTION
**Pull request by alexca1703**
Finished step 13 for mochawesome reports install and configuration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,21 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
+    "clean:reports": "rm -rf cypress/reports && mkdir cypress/reports",
+    "clean:reportsWindows": "rmdir /S /Q cypress\\reports && mkdir cypress\\reports",
```
Comment (holgiosalos) [inline package.json]: if you use the git bash console in windows you will not need this command

File: package.json [hunk 2]
```diff
@@ -4,10 +4,21 @@
...
     "test": "cypress run --headless --browser chrome",
+    "testWindows": "cypress run --headless --browser chrome",
```
Comment (holgiosalos) [inline package.json]: if you use the git bash console in windows you will not need this command

File: package.json [hunk 3]
```diff
@@ -4,10 +4,21 @@
...
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "combine:reportsWindows": "mochawesome-merge cypress\\reports\\mocha\\*.json > cypress\\reports\\report.json",
```
Comment (holgiosalos) [inline package.json]: if you use the git bash console in windows you will not need this command

File: package.json [hunk 4]
```diff
@@ -4,10 +4,21 @@
...
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports",
+    "generate:reportsWindows": "marge cypress\\reports\\report.json -f report -o cypress\\reports",
```
Comment (holgiosalos) [inline package.json]: if you use the git bash console in windows you will not need this command

File: package.json [hunk 5]
```diff
@@ -4,10 +4,21 @@
...
+    "pretest": "npm run clean:reports",
+    "pretestWindows": "npm run clean:reportsWindows",
```
Comment (holgiosalos) [inline package.json]: if you use the git bash console in windows you will not need this command

File: package.json [hunk 6]
```diff
@@ -4,10 +4,21 @@
...
+    "posttest": "npm run combine:reports && npm run generate:reports",
+    "posttestWindows": "npm run combine:reportsWindows && npm run generate:reportsWindows"
```
Comment (holgiosalos) [inline package.json]: if you use the git bash console in windows you will not need this command

File: package.json [hunk 7]
```diff
@@ -4,10 +4,21 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
+    "clean:reports": "rm -rf cypress/reports && mkdir cypress/reports",
```
Comment (holgiosalos) [inline package.json]: you can change this to: `rm -rf cypress/reports/* || true` so in that way the rem -rf will not fail when reports dir does not exist

File: package.json (hunk 1 above)
Comment (alexca1703) [reply package.json]: Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos

File: package.json (hunk 3 above)
Comment (alexca1703) [reply package.json]: Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos

File: package.json (hunk 5 above)
Comment (alexca1703) [reply package.json]: Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos

File: package.json (hunk 6 above)
Comment (alexca1703) [reply package.json]: Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos

File: package.json (hunk 4 above)
Comment (alexca1703) [reply package.json]: Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos

File: package.json (hunk 2 above)
Comment (alexca1703) [reply package.json]: Tested with git bash after removing windows command. worked as expected. thanks @holgiosalos

**Review by holgiosalos — APPROVED** [review]

File: package.json (hunk 7 above)
Comment (alexca1703) [reply package.json]: I've updated command and it worked as expected with git bash. Thanks @holgiosalos

**Pull request merged**